package.json

```diff
@@ -37,15 +37,18 @@
   "homepage": "https://github.com/finnishtransportagency/ratatiedot-extranet#readme",
   "dependencies": {
     "aws-cdk-lib": "^2.45.0",
+    "axios": "^1.1.3",
     "concurrently": "^7.4.0",
-    "constructs": "^10.0.0"
+    "constructs": "^10.0.0",
+    "moment": "^2.29.4",
```

Comment:
These days, it would be better not to adopt `moment`, as seen here: https://momentjs.com/docs/ I don't have experience with any of the alternatives, but now's a good chance to gain some 😉